src/source/Signaling/Client.c

```diff
@@ -1,8 +1,10 @@
 #define LOG_CLASS "SignalingClient"
 #include "../Include_i.h"
+#include "com/amazonaws/kinesis/video/utils/Include.h"
```

Comment:
This is not required. `Include_i.h` uses this file anyways.

Reply:
done